replications/internal/queue_management.go

```diff
@@ -141,7 +140,7 @@ func (rq *replicationQueue) run() {
 // SendWrite is responsible for processing all data in the queue at the time of calling.
 // Retryable errors should be handled and retried in the dp function.
 // Unprocessable data should be dropped in the dp function.
-func (rq *replicationQueue) SendWrite(dp func([]byte) error) bool {
+func (rq *replicationQueue) SendWrite() (error bool) {
```

Comment:
Is there a reason we named this return `bool`? Or is it just for clarity

Reply:
Nope, I think that was just an oversight. Seems pretty clear without having the named return so I'll clean that up!